flink-connectors/flink-connector-cassandra/src/main/java/org/apache/flink/streaming/connectors/cassandra/CassandraAppendTableSink.java

```diff
@@ -80,9 +81,16 @@ public void emitDataStream(DataStream<Row> dataStream) {
 			CassandraSink.addSink(dataStream)
 				.setClusterBuilder(this.builder)
 				.setQuery(this.cql)
-				.build();
+				.build()
+				.name(explainSink());
 		} catch (Exception e) {
 			throw new RuntimeException(e);
 		}
 	}
+
+	@Override
+	public String explainSink() {
+		return getClass().getSimpleName() + " "
```

Comment:
I would move the expression to a util class because it is used for all other sinks and sources as well. And also remove the space. Btw fieldNames can be null if the sink is not configured. So maybe we should use something like "CassandraAppendTableSink(*)" in this case.